wormhole/Dockerfile

```diff
@@ -1,14 +1,12 @@
-FROM pypy:3.9-7.3.9-bullseye
-
-COPY Pipfile /app/Pipfile
-COPY Pipfile.lock /app/Pipfile.lock
-
-RUN apt update
-RUN apt install rustc -y
+# Pull official base image from DockerHub
+FROM python:3.9.16-alpine3.17
```

Comment:
Or maybe just `3.9-alpine`? (see [my other comment](https://github.com/LeastAuthority/magic-wormhole-docker/pull/21#discussion_r1187518065))

Reply:
See my other comment too: I'd rather making it explicit, since we need to merge and rebuild for minor update anyway.

Reply:
More on this in #29